### PR TITLE
Add English localization file

### PR DESCRIPTION
**Pull request by chk1**

### CODE REVIEW

File: src/BackupBasic_en_US.properties [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Error = Error
+Details = Additional details are in the console
+CannotCreateFile = An error occurred while creating the following file:
+
+//Logger(Main)
+LogError = An error occurred while creating the log file!
+LogStart = Logger started.
+
+//ArgParser
+Usage = Usage: Backup.jar SourcePath(String or default) DestPath(String or default) [ChecksumOldDir] (true (Standard), false) [ChecksumFinished] (true (Standard), false).
```
Comment (ichttt) [inline src/BackupBasic_en_US.properties]: Please change "Standard" to "default"

**Review by ichttt — CHANGES_REQUESTED** [review]
One minor thing, otherwise fine.

File: src/BackupBasic_en_US.properties (hunk 1 above)
Comment (chk1) [reply src/BackupBasic_en_US.properties]: Done

**Review by ichttt — APPROVED** [review]

**Comment by ichttt** [comment]
thanks!

**Pull request merged**